src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -341,5 +346,49 @@ private string GetValidFileName(string fileName)
             fileName = Uri.EscapeDataString(fileName);
             return fileName;
         }
+
+        private List<string> GetOptionIdsFromFormLayout(string formLayout)
```

Comment:
Could this potentially be moved to a helper class?

Reply:
Yeah, I guess, but I like having the implementation close to where it is used. I don't think any other classes would require to reuse this exact functionality.

Reply:
Fair enough. Approved :)